### PR TITLE
Add the install path for Homebrew in the ARM macOS

### LINKED ISSUE #222: Mac OS配置中添加软链接的命令不能执行
````
$ cd /usr/local/bin/
$ ln -s gcc-10 gcc
$ ln -s g++-10 g++
````
文中上述内容执行后报错，intel和arm的Mac都是报错

### PR DESCRIPTION
**Pull request by core-man**
Fixes #222

### CODE REVIEW

**Comment by core-man** [comment]
Ping @wangliang1989 for the review. BTW, would you like to join the [seismo-learn contributor](https://github.com/orgs/seismo-learn/teams/contributors) team?

File: source/best-practices/software-installation.rst [hunk 1]
```diff
@@ -84,10 +85,10 @@ Linux 或 macOS 系统下安装软件通常有如下几种方式：
 这里推荐将科研相关软件安装到 :file:`~/opt/` 目录下，而不是更常见的 :file:`/opt/`
 或 :file:`/usr/local/` 目录，主要是基于如下几点考虑：
 
--   某些包管理器使用了这些目录，比如 macOS 下的 Homebrew 的软件安装路径是 :file:`/usr/local/`\ ，
+-   某些包管理器使用了这些目录，比如非 ARM 架构 macOS 下的 Homebrew 的软件安装路径是 :file:`/usr/local/`\ ，
     macOS 的 MacPorts 的软件安装路径是 :file:`/opt/local/`
 -   某些闭源或商业软件（如 Matlab、Google Chrome、Google Earth）默认将软件安装到
-    :file:`/opt/` 目录下
+    :file:`/opt/` 目录下。ARM 架构的 macOS 下的 Homebrew 也被安装到该目录下
```
Comment (core-man) [inline source/best-practices/software-installation.rst:91]:
```suggestion
    :file:`/opt/` 目录下。ARM 架构 macOS 下的 Homebrew 也被安装到该目录下
```

**Review by seisman — APPROVED** [review]

File: source/computer/macos-setup.rst [hunk 2]
```diff
@@ -95,6 +95,11 @@ Homebrew 会被安装到 :file:`/usr/local/` 目录下。通过 Homebrew 安装
    同样由于网络的原因，Homebrew 在国内可能下载速度较慢。建议参照
    https://github.com/ineo6/homebrew-install 教程设置中科大源或清华大学源。
 
+.. note::
+
+   在最新 ARM 架构的 macOS 下，Homebrew 会被安装到 :file:`/opt/homebrew` 目录下。
```
Comment (core-man) [inline source/computer/macos-setup.rst:100]:
```suggestion
   在最新 ARM 架构的 macOS 下，Homebrew 会被安装到 :file:`/opt/homebrew/` 目录下。
```

**Comment by wangliang1989** [comment]
我觉得没问题

**Pull request merged**